Domaciukoly20/driving_cars.py

```diff
@@ -0,0 +1,54 @@
+import time
+
+class Car:
+    def __init__(self, economy, color, model, mileage=0, fuel=100, **kwargs):
+        super().__init__(**kwargs)
+        self._mileage = mileage
+        self._fuel = fuel
+        self._economy = economy
+        self._color = color
+        self._model = model
+
+    def drive(self, distance):
+        for i in range(int(distance)):
```

Comment:
if your temp-variable (`i`) is not used in the loop then it's better to use `_` instead of `i`